### PR TITLE
Employees metadata get e ps

### PR DESCRIPTION
**Pull request by Sasporta**

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -7,6 +7,6 @@ export default async (): Promise<Config.InitialOptions> => {
     collectCoverage: true,
     testEnvironment: 'node',
     coverageReporters: ['text-summary'],
-    collectCoverageFrom: ['src/controllers/**'],
+    collectCoverageFrom: ['src/controllers/**', 'src/modules/**'],
```
Comment (idankepten) [inline jest.config.ts:10]: why not 'src/**' ? 
every folder you need to add here

File: src/controllers/employees/delete.ts [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 
-export const deleteEmployee = async ({ params: { id: uuid } }: Request) => {
-  await EmployeeModule.destroy(uuid);
+export const deleteEmployee: RouteHandler = async (
+  { params: { id: uuid } },
+  res,
+  next,
+) => {
+  try {
+    await EmployeeModule.destroy(uuid);
```
Comment (idankepten) [inline src/controllers/employees/delete.ts:11]: please think (don't change) why this await is not in the Promise.all ?

File: src/controllers/employeesMetadata/getAll.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  next,
+) => {
+  try {
+    const whereStatement = companyUuid ? { companyUuid } : {};
+
+    const resultsLimit = EmployeeMetadataModule.limit(+limit);
+
+    const employees = await EmployeeMetadata.find(whereStatement).limit(
+      resultsLimit,
+    );
```
Comment (idankepten) [inline src/controllers/employeesMetadata/getAll.ts:17]: 💯 

just remove the empty lines

File: jest.config.ts (hunk 1 above)
Comment (Sasporta) [reply jest.config.ts:10]: not everything:
config folder doesnt need tests - therefore should be in the coverage

same for entities and models, seeds, types etc...

I'm covering only the files I need to test - then if its not 100% I can actually see what I missed and add test / decide not to test it for some reason.

if ill cover everything the % wont tell me anything...

File: src/controllers/employeesMetadata/getAll.ts (hunk 3 above)
Comment (Sasporta) [reply src/controllers/employeesMetadata/getAll.ts:17]: never

**Pull request merged**